lisa/transformer.py

```diff
@@ -71,6 +73,30 @@ def _internal_run(self) -> Dict[str, Any]:
         raise NotImplementedError()
 
 
+class DeploymentTransformer(Transformer):
```

Comment:
Try to define it like a Mixins pattern.

Reply:
It's the only one comment.

Reply:
I moved into transformers folder. Can you clarify on this comment?

Reply:
Try to write it as a Mixin pattern like InitializableMixin. So, when there are more inheriting on transformers, it could be more flexible.

Reply:
Since I am overriding __init__ to take in an extra parameter, I'm not sure how well it fits as a Mixin class. I don't want the implementation to require a specific order for multi-class inheritance.
I agree with you that this could be made more flexible, but I'm not sure the best way to implement it.

Reply:
ok, we can go with this.